core/src/main/scripts/importer/validateData.py

```diff
@@ -4025,9 +4038,15 @@ def interface(args=None):
                                    action='store_true',
                                    help='Skip tests requiring information '
                                         'from the cBioPortal installation')
-    parser.add_argument('-P', '--portal_properties', type=str,
```

Comment:
okay, so looks like `portal_properties` was used exclusively for loading gene info only? (which will now be provided optionally as input arguments)

Reply:
validateData script no longer uses portal.properties file.